schema/plugin.json

```
@@ -0,0 +1,148 @@
+{
+  "title": "Theme editor",
+  "description": "Settings of Theme Editor.",
+  "type": "object",
+  "properties": {
+    "useSettings": {
+      "type": "boolean",
+      "title": "Use the settings style",
+      "default": true
+    },
+    "customStyles": {
+      "title": "Tunable CSS properties",
+      "type": "object",
+      "properties": {
```

There is a bug in a dependency that prevent the default value for `customStyles` to be found.

Would you mind adding a `default` here that gather all keys default values:

```suggestion
      "default" : {
        "layout-color": "#111111",
        // etc
      },
      "properties": {
```

---

@fcollonval Ok thanks. This has been included.